src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -77,97 +92,68 @@ def split_tensor_along_last_dim(tensor, num_partitions, contiguous_split_chunks=
     return tensor_list
 
 
-def attention_mask_func(attention_scores, attention_mask, causal_mask):
-    attention_mask_bool = ~attention_mask.bool()
+def _expand_mask(mask: torch.Tensor, dtype: torch.dtype, tgt_len: int = None):
+    """
+    Expands attention_mask from `[bsz, seq_len]` to `[bsz, 1, tgt_seq_len, src_seq_len]`.
+    """
+    bsz, src_len = mask.size()
+    tgt_len = tgt_len if tgt_len is not None else src_len
```

Comment:
Same, use better names here please.

Reply:
Fixed in : 563c3234228a0878ff79bd58fd6042c2211c323a